src/main/java/fr/uge/chargepointconfiguration/chargepoint/ocpp/ocpp16/OcppMessageBuilder16.java

```diff
@@ -0,0 +1,31 @@
+package fr.uge.chargepointconfiguration.chargepoint.ocpp.ocpp16;
+
+import fr.uge.chargepointconfiguration.chargepoint.WebSocketRequestMessage;
+import fr.uge.chargepointconfiguration.chargepoint.ocpp.OcppMessage;
+import fr.uge.chargepointconfiguration.chargepoint.ocpp.OcppMessageBuilder;
+import fr.uge.chargepointconfiguration.chargepoint.ocpp.RegistrationStatus;
+import fr.uge.chargepointconfiguration.tools.JsonParser;
+import java.time.LocalDateTime;
+
+/**
+ * Builds an OCPP 1.6 message.
+ */
+public class OcppMessageBuilder16 implements OcppMessageBuilder {
+
+  @Override
+  public OcppMessage buildMessage(WebSocketRequestMessage webSocketRequestMessage) {
+    return switch (webSocketRequestMessage.messageName()) {
+      case "BootNotification" ->
```

Comment:
"BootNotification" may be in an enum, that can help the developper know what kind of message can be built/parsed I guess

Reply:
I like your idea, this way, we don't handle strings like dummies.
Will do

Reply:
Done